src/gg.regression.unity.bots/Runtime/Scripts/StateRecorder/BotSegments/Models/CVObjectDetectionKeyFrameCriteriaData.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Text;
+using System.Threading;
+using JetBrains.Annotations;
+using RegressionGames.StateRecorder.BotSegments.JsonConverters;
+using RegressionGames.StateRecorder.JsonConverters;
+
+namespace RegressionGames.StateRecorder.BotSegments.Models
+{
+    [Serializable]
+    public class CVObjectDetectionKeyFrameCriteriaData : IKeyFrameCriteriaData
+    {
+        // update this if this schema changes
+        public int apiVersion = SdkApiVersion.VERSION_9;
```

Comment:
Is this the latest version? There may be a newer one, not sure.

Reply:
That definitely was not the newest one. Updated to the newest of 15.